### PR TITLE
Fix version and caps

### PR DESCRIPTION
**Pull request by Ankit098**
- Report correct version of service.
- Add wdioService cap correctly in edge case.

### CODE REVIEW

File: src/launcher.ts [hunk 1]
```diff
@@ -28,9 +31,34 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
     onPrepare (config?: Options.Testrunner, capabilities?: Capabilities.RemoteCapabilities) {
         if (Array.isArray(capabilities)) {
             capabilities.forEach((capability: Capabilities.DesiredCapabilities | any) => {
-                const wdioServiceVersion = process.env.npm_package_version;
+                const wdioServiceVersion = version;
+                let webdriverIOVersion: any = undefined;
+                const packageFile: any = process.env.npm_package_json;
+                if (packageFile !== undefined) {
```
Comment (SaranshBS) [inline src/launcher.ts]: Lets organize this, put the newer code in a new method

File: src/launcher.ts [hunk 2]
```diff
@@ -28,9 +31,34 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
     onPrepare (config?: Options.Testrunner, capabilities?: Capabilities.RemoteCapabilities) {
         if (Array.isArray(capabilities)) {
             capabilities.forEach((capability: Capabilities.DesiredCapabilities | any) => {
-                const wdioServiceVersion = process.env.npm_package_version;
+                const wdioServiceVersion = version;
+                let webdriverIOVersion: any = undefined;
+                const packageFile: any = process.env.npm_package_json;
```
Comment (SaranshBS) [inline src/launcher.ts]: `const packageFile: any = process.env.npm_package_json;`
Wont this point to package.json of the repo instead of lib?

**Review by SaranshBS — CHANGES_REQUESTED** [review]
Please add comments wherever necessary

File: src/launcher.ts (hunk 2 above)
Comment (Ankit098) [reply src/launcher.ts]: yes, intended. Want to get the webdriverio version used in repo.

File: src/launcher.ts (hunk 1 above)
Comment (Ankit098) [reply src/launcher.ts]: done

**Review by SaranshBS — APPROVED** [review]

**Pull request merged**